bundles/org.openhab.binding.tapocontrol/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -37,6 +37,22 @@
 		</channels>
 	</channel-group-type>
 
+	<channel-group-type id="powerStrip">
+		<label>SmartPlug</label>
+		<description>Tapo Smart Plug Power Outlet</description>
+		<channels>
+			<channel id="output1" typeId="outputChannel">
+				<label>Output Switch #1</label>
```

Comment:
Consider also providing (overriding) descriptions mentioning 1, 2 and 3. Is "#" needed here, or simply:
```suggestion
				<label>Output Switch 1</label>
```

?

Reply:
Done

Reply:
Please regenerate I18N after this change.

Reply:
Done.